client/jetpack-connect/controller.js

```diff
@@ -359,3 +359,25 @@ export function sso( context, next ) {
 	);
 	next();
 }
+
+export function authorizeOrSignup( context, next ) {
+	const isLoggedIn = isUserLoggedIn( context.store.getState() );
+
+	if ( isLoggedIn ) {
+		authorizeForm( context, next );
+		return;
+	}
+
+	signupForm( context, next );
+}
+
+export function redirectToLoginIfLoggedOut( context, next ) {
+	const isLoggedIn = isUserLoggedIn( context.store.getState() );
+
+	if ( ! isLoggedIn ) {
+		page( login( { isNative: true, isJetpack: true, redirectTo: context.path } ) );
```

Comment:
I'm not seeing the `isNative` parameter referenced anywhere else in the codebase. Am I missing something, or is this safe to remove?

Reply:
Nice catch! `isNative` was removed in #53191 so it seems like this is an artifact from a bad rebase. Removed in da6d8a8358b7f277396e5721ce8ededfecd296a6